quattor/types/system.pan

```diff
@@ -27,7 +27,7 @@ type structure_oldname = {
 include 'components/network/core-schema';
 
 # TODO: monitoring to be defined
-include 'monitoring/lemon/schema';
+include { if_exists('monitoring/lemon/schema') };
```

Comment:
Does it even need a variable?
Can we just leave it up to sites to include an appropriate schema for `/system/monitoring` if they want one?